src/onegov/reservation/models/custom_allocation.py

```diff
@@ -5,7 +5,7 @@
 
 
 class CustomAllocation(Allocation, ModelBase):
-    __mapper_args__ = {'polymorphic_identity': 'custom'}  # type:ignore
+    __mapper_args__ = {'polymorphic_identity': 'custom'}
```

Comment:
Another merge conflict resolution error, this ignore is necessary on latest libres version.